spread.yaml

```diff
@@ -66,18 +154,26 @@ prepare: |
 
   apt install -y snapd
 
-  # If $SNAPCRAFT_CHANNEL is defined, install snapcraft from that channel.
-  # Otherwise, look for it in /snapcraft/.
-  if [ -z "$SNAPCRAFT_CHANNEL" ]; then
-      if stat /snapcraft/*.snap 2>/dev/null; then
-          snap install --classic --dangerous /snapcraft/*.snap
+  if [ "$SNAPCRAFT_PACKAGE_TYPE" = "deb" ]; then
```

Comment:
maybe not now, but this might be nice to have in helper scripts to be able to use shellcheck? Or did we discuss this already and am I going on in circles

Reply:
I agree 100%, sadly, see [this comment](https://github.com/snapcore/snapcraft/pull/2188/files/909bef12452e1022faf774ed3c609a937c577679#diff-3c11e56e5f7f82b0f352d0fe1851a107R129) for why it has to be this way.